### DISCUSSION TITLE
Thanks

### DISCUSSION THREAD

**Discussion by redfredc** [codewars/codewars.com#2649] [General]
Can you provide more details?

**Comment by hobovsky** [comment]
Could you provide a link to your Codewars profile, or your Codewars user name?

**Reply by redfredc** [reply]
fredc
https://www.codewars.com/users/fredc/comments

**Reply by hobovsky** [reply]
As far as I can see, the account got suspended because it submitted a significant amount of copied solutions, including purple kata.

If you need any specific details or further explanations, I'd suggest talking to a moderator in `#support-requests` channel of Codewars Discord.
